modules/combined/test/tests/thermal_elastic/tests

```diff
@@ -1,25 +1,59 @@
 [Tests]
-  design = 'ThermalConductivity.md'
-  issues = '#13232'
   [./test]
     type = 'Exodiff'
     input = 'thermal_elastic.i'
     exodiff = 'thermal_elastic_out.e'
+    design = 'ComputeVariableIsotropicElasticityTensor.md'
+    issues = '#8265'
     requirement = "The system shall compute constant stress in the xx, yy, zz, xy, yz, and xz direction
                    using irregular elements with functional Poisson's ratios and Young's moduli."
   [../]
+  [./ad-test]
+    type = 'Exodiff'
+    input = 'ad-thermal_elastic.i'
+    exodiff = 'ad-thermal_elastic_out.e'
+    design = 'ADComputeVariableIsotropicElasticityTensor.md'
+    issues = '#13232'
+    requirement = "The system shall compute constant stress in the xx, yy, zz, xy, yz, and xz direction
+                   using irregular elements with functional Poisson's ratios and Young's moduli."
+  [../]
+  [./ad-test-jac]
+    type = 'PetscJacobianTester'
+    input = 'ad-thermal_elastic.i'
+    petsc_version = '>=3.9.4 || <3.9.0'
+    ratio_tol = 6e-7
+    method = 'OPT'
```

Comment:
Why only opt mode for these tests?

Reply:
dbg takes too long. since it's a jacobian evaluation, and there is already a opt/dbg test for just running, I restricted it.